CruzHacks/cruzhacks-2021-website
# Fixed Failing Mailing Service

Problem & Solution
=======
The [mailing service function](https://github.com/CruzHacks/cruzhacks-mailing-microservice) requires an API key to go in the header of the request to authenticate it. We accidentlly overlooked this crucial step, which is the reason why we were getting a `401 UNAUTHORIZED` message. I went ahead and added a `REACT_APP_API_KEY` to the header of the request and we got it to work! 

Steps to Verify:
----------------
1. Spin up a local version of the mailing microservice 
1. Spin up a local version of our web app 
1. Try and sign `dev@cruzhacks.com` up for the emailing service. Verify that you get a `200 OK` from the backend with a message saying that the user already exists.

Screenshots:
-----------------------
<img width="533" alt="Screen Shot 2020-11-01 at 8 30 03 PM" src="https://user-images.githubusercontent.com/40477553/97830723-c7e60980-1c82-11eb-81a5-b7f17015a52d.png">
